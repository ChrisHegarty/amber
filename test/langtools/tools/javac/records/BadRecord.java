/*
 * @test /nodynamiccopyright/
 * @summary Verifying error recovery for broken record classes
 * @compile/fail/ref=BadRecord.out --enable-preview -source ${jdk.version} -XDrawDiagnostics BadRecord.java
 */
record BadRecord001 {}

record BadRecord0022( {}

record BadRecord0033(int {}

record BadRecord0044(int) {}

record BadRecord005(int i {}

record BadRecord006(int i, {}

record BadRecord007(int i,) {}

record BadRecord008(int i, int {}

record BadRecord009(int i, int) {}

record BadRecord010(int i, int j {}

record BadRecord011(int i, int j, {}

record BadRecord012(int i, int j,) {}

record BadRecord013;

record BadRecord014(;

record BadRecord015(int;

record BadRecord016(int);

record BadRecord017(int i;

record BadRecord018(int i,;

record BadRecord019(int i,);

record BadRecord020(int i, int;

record BadRecord021(int i, int);

record BadRecord022(int i, int j;

record BadRecord023(int i, int j,;

record BadRecord024(int i, int j,);

record BadRecord025(int x)

record BadRecord026 {}

record BadRecord027(final int x) { }

record BadRecord028(private int x) { }

record BadRecord029(public int x) { }

record BadRecord030(volatile int x) { }

record BadRecord030(int x) {
    private int x() { return x; }
}

record R(int x) {
    public int x;
}
