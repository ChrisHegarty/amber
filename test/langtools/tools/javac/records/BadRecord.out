BadRecord.java:6:20: compiler.err.expected: '('
BadRecord.java:6:21: compiler.err.illegal.start.of.type
BadRecord.java:8:23: compiler.err.illegal.start.of.type
BadRecord.java:10:25: compiler.err.expected: token.identifier
BadRecord.java:12:25: compiler.err.expected: token.identifier
BadRecord.java:14:26: compiler.err.expected3: ',', ')', '['
BadRecord.java:16:28: compiler.err.illegal.start.of.type
BadRecord.java:18:27: compiler.err.illegal.start.of.type
BadRecord.java:20:31: compiler.err.expected: token.identifier
BadRecord.java:22:31: compiler.err.expected: token.identifier
BadRecord.java:24:33: compiler.err.expected3: ',', ')', '['
BadRecord.java:26:35: compiler.err.illegal.start.of.type
BadRecord.java:28:34: compiler.err.illegal.start.of.type
BadRecord.java:30:20: compiler.err.expected: '('
BadRecord.java:32:21: compiler.err.illegal.start.of.type
BadRecord.java:34:24: compiler.err.expected: token.identifier
BadRecord.java:36:24: compiler.err.expected: token.identifier
BadRecord.java:36:25: compiler.err.expected: '{'
BadRecord.java:38:26: compiler.err.expected3: ',', ')', '['
BadRecord.java:40:27: compiler.err.illegal.start.of.type
BadRecord.java:42:27: compiler.err.illegal.start.of.type
BadRecord.java:42:28: compiler.err.expected: '{'
BadRecord.java:44:31: compiler.err.expected: token.identifier
BadRecord.java:46:31: compiler.err.expected: token.identifier
BadRecord.java:46:32: compiler.err.expected: '{'
BadRecord.java:48:33: compiler.err.expected3: ',', ')', '['
BadRecord.java:50:34: compiler.err.illegal.start.of.type
BadRecord.java:52:34: compiler.err.illegal.start.of.type
BadRecord.java:52:35: compiler.err.expected: '{'
BadRecord.java:54:27: compiler.err.expected: '{'
BadRecord.java:58:21: compiler.err.record.cant.declare.field.modifiers
BadRecord.java:60:21: compiler.err.record.cant.declare.field.modifiers
BadRecord.java:62:21: compiler.err.record.cant.declare.field.modifiers
BadRecord.java:64:21: compiler.err.record.cant.declare.field.modifiers
BadRecord.java:71:17: compiler.err.record.cannot.declare.instance.fields
BadRecord.java:72:2: compiler.err.premature.eof
36 errors
